reporter/reporter.go

```diff
@@ -97,7 +97,12 @@ func (r *reporter) ReportCounter(id id.ID, value int64) error {
 	multiErr := xerrors.NewMultiError()
 	matchRes := r.matcher.Match(id)
 	earliestNanos := r.nowFn().Add(-r.maxNegativeSkew).UnixNano()
-	if err := r.server.WriteCounterWithPoliciesList(id.Bytes(), value, matchRes.MappingsAt(earliestNanos)); err != nil {
+	if err := r.server.WriteCounterWithPoliciesList(
+		id.Bytes(),
+		value,
+		// NB(cw) We do not support custom aggregation types for Counter in mapping rules.
```

Comment:
Yeah that's probably cleaner on the backend, in case we do need to support it in the future, we don't have to change code in the backend.